mitodl/bootcamp-ecommerce
# Implement take video link

#### What are the relevant tickets?
Part of #604

#### What's this PR do?
 - "Take Video Interview" link which opens the drawer
 - "Take Interview At Jobma" link which creates a Jobma interview, then redirects the user to that page
 - "View Your Interview" link after the Jobma interview is complete. Note that you won't get a postback from Jobma on your local instance.

#### How should this be manually tested?
 - Set up Jobma environment variables (ask me for them if you don't have these already)
 - This can be tricky, but try to get your application to `AWAIT_USER_SUBMISSIONS` and make sure there aren't any current submissions. 
 - Go to `/applications`. You should see a list of applications. Expand the one you've been editing and you should see a `Take Video Interview` link. Click that and the drawer should open.
 - Click `Take Interview at Jobma` and you should be redirected there after a brief delay. You don't need to actually take the Jobma interview, just being redirected there is enough.
 - Find the `VideoInterviewSubmission` related to the bootcamp (there's likely only one since no other code should create it). Find the `AppStepSubmission` related to it. Set its `submission_status` to `'submitted'` and save. Also find the `Interview` related to `VideoInterviewSubmission`, and set its `results_url` to whatever valid URL you prefer.
 - View the application again and you should see a `View Your Interview` link. That link should point to the URL you set in the previous step.

#### Screenshots
![Screenshot from 2020-06-18 09-56-14](https://user-images.githubusercontent.com/863262/85029167-fcc94800-b149-11ea-8298-e645717cd1c7.png)

![Screenshot from 2020-06-18 09-58-33](https://user-images.githubusercontent.com/863262/85029466-4e71d280-b14a-11ea-86de-dad5b470bfba.png)
